gemfire-core/src/main/java/com/gemstone/gemfire/internal/LinuxProcFsStatistics.java

```diff
@@ -236,14 +236,14 @@ private static void getLoadAvg(double[] doubles, int[] ints) {
       if (threadLoad != null) {
         int slashIndex = threadLoad.indexOf('/');
         if (slashIndex >= 0) {
-          try {
-            ints[LinuxSystemStats.threadsINT] = Integer.parseInt(
-                threadLoad.substring(slashIndex + 1));
-          } catch (NumberFormatException ignored) {
-          }
+          threadLoad = threadLoad.substring(slashIndex + 1);
+        }
+        try {
+          longs[LinuxSystemStats.totalThreadsLONG] = Long.parseLong(threadLoad);
+        } catch (NumberFormatException ignored) {
```

Comment:
If the threadLoad is high and the number before the '/' is the number actually scheduled and if it is less than the number of cpu available then does this mean less cpu utilization. Should this be captured too. If this is consistent then we can say that the cpu is not being optimally utilized.